budgetportal/tests/test_guides_pages.py

```diff
@@ -33,5 +33,12 @@ def test_guide_page(self):
         """Simple test of template response for guide page"""
         response = Client().get(self.guide_page.url_path)
         self.assertContains(response, self.guide_page.title)
+
+        image_url_without_params = self.guide_page.image.file.url.split("?")[0]
+        # those modifications are done dynamically
+        image_url = image_url_without_params.replace("original_", "")
+        image_url = image_url.replace(".jpg", ".max-320x200.jpg")
+        self.assertContains(response, image_url)
```

Comment:
rather hardcode the image URL and check that it's present in the response, e.g the rough guess below.
Please take the comment varbatim


```suggestion
        ## Verify the integration of our configuration, django-storages and wagtail
        ## - that is - the generated image URL templated in matches the configuration of the site
        self.assertContains(response, "http://minio:9000/images/some-image.max-320x200.jpg")
```